packages/main/src/ColorPalette.js

```diff
@@ -58,6 +84,15 @@ const metadata = {
 		_selectedColor: {
 			type: CSSColor,
 		},
+
+		/**
+		 * Defines if the palette is in Popup or Embeded mode.
+		 * @type {CSSColor}
+		 * @private
+		 */
+		popupMode: {
```

Comment:
This property is never assigned, but it is used?

Reply:
It is assigned in the color-palette-popover? And it's default value is false.